Mathlib/Order/Basic.lean

```diff
@@ -1189,6 +1189,9 @@ namespace Prod
 instance (α : Type u) (β : Type v) [LE α] [LE β] : LE (α × β) :=
   ⟨fun p q ↦ p.1 ≤ q.1 ∧ p.2 ≤ q.2⟩
 
+instance (α : Type u) (β : Type v) [LE α] [LE β] [@DecidableRel α (·≤·)] [@DecidableRel β (·≤·)] :
+    @DecidableRel (α × β) (·≤·) := fun _ _ => And.decidable
```

Comment:
This is possibly not general enough; we should probably have instead
```suggestion
instance (α : Type u) (β : Type v) [LE α] [LE β] (x y : α × β) [Decidable (x.1 ≤ y.1)] [Decidable (x.2 ≤ y.2)] :
    DecidableRel (x ≤ y) := And.decidable
```
(not tested)

Reply:
Yes, I see, that is indeed a lot more general